src/routes/index.ts

```diff
@@ -8,48 +9,106 @@ import {
   renameFilesToChecksum,
 } from 'middlewares/files'
 import path from 'path'
+import qs from 'querystring'
+import redis from 'redis'
 import logger from 'utils/logger'
 
 const router = express.Router()
 
+const redisClient = redis.createClient({ url: process.env.REDIS_URI })
+
+const DEFAULT_EXPIRE = Number(process.env.CACHE_EXPIRE || 60)
+
+const cache = ExpressRedisCache({
+  client: redisClient,
+  prefix: 'file',
+  expire: DEFAULT_EXPIRE, // 1 min,
+})
+
+cache.on('message', message => logger.info('Cached %s', message))
```

Comment:
I think we should use verbose in cache events because the hudge amounts of message will stress the CPU in production.

```suggestion
cache.on('message', message => logger.verbose('Cached %s', message))
```